torchgeo/models/dofa.py

```diff
@@ -518,10 +524,12 @@ def dofa_huge_patch16_224(**kwargs: Any) -> DOFA:
     .. versionadded:: 0.6
 
     Args:
+        *args: Additional arguments to pass to :class:`DOFA`.
         **kwargs: Additional keywork arguments to pass to :class:`DOFA`.
 
     Returns:
         A DOFA huge 16 model.
     """
-    model = DOFA(patch_size=14, embed_dim=1280, depth=32, num_heads=16, **kwargs)
+    kwargs |= {'patch_size': 14, 'embed_dim': 1280, 'depth': 32, 'num_heads': 16}
```

Comment:
Should we warn the user if these overwrite kwargs?

Reply:
I think it's worth adding, just not in this PR. Torchvision does issue a warning as of https://github.com/pytorch/vision/pull/5618.